common/bin/oo-diagnostics

```diff
@@ -1204,6 +1204,39 @@ class OODiag
 
   end
 
+  def test_auth_keys_salt_failure
+    skip_test unless @is_broker
+
+    log_dir = "/var/log/openshift/broker/"
+
+    if Rails.env.development?
+      mode = "development"
+      msg_log = `grep 'Broker key authentication failed' #{log_dir + mode}.log`
+    elsif Rails.env.production?
+      mode = "production"
+      msg_log = `grep 'Broker key authentication failed' #{log_dir + mode}.log`
+    else
+      mode = "test"
+      msg_log = `grep 'Broker key authentication failed' #{log_dir + mode}.log`
```

Comment:
The `msg_log =` line does not need to be repeated three times. Place it only once after the `if...end` statement since it is the same in each case.